snapshots/energy_institute/2023-06-26/statistical_review_of_world_energy.csv.dvc

```diff
@@ -0,0 +1,23 @@
+meta:
+  name: Energy Institute Statistical Review of World Energy 2023
+  publication_year: 2023
+
```

Comment:
I don't think it matters at all but the line break here seems unusual (I assume it is automatically added?)